### ISSUE TITLE
 four cameras with fixed relative postures，Improve Accuracy？

### ISSUE THREAD

**Issue by pzh11001** [liyi14/mx-DeepIM#39]
One of the reasons for the inaccuracy mentioned in your paper is that using only one view of the image can easily lead to inaccurate Z-direction estimation, resulting in large deviations. I wonder if I can use multiple cameras from different angles.

I also tried to use four cameras with fixed relative postures. Four 640x480 images were stitched in the long and wide directions, and turned into a 1280x960 image. Then it is sent to the network for learning, but the final learning effect is very poor. The 2rd accuracy of ape model is only 70% in testing. I have trained 14 rounds, 12 iterations, Adam optimizer, learning rate 5e-6.

I've thought about it, because the principle of this network is based on optical streaming network. Will it be possible for the network to incorrectly match the image of the second camera with the image of the first camera when there are multiple camera pictures? But this is just an idea.

Do you have any improvements in this area?

**Comment by wangg12** [comment]
Perhaps you need to check the zoom in operation performed on your stitched images.

**Comment by pzh11001** [comment]
Thank you very much for your answer. It seems that it should be this question I'll try to improve the code.
